source/fundamentals/connection/connection-options.txt

```diff
@@ -0,0 +1,418 @@
+       |
+       | **Data Type**: ``ConnectionStringScheme``
+       | **Default**: ``ConnectionStringScheme.MongoDB``
+      
+   * - **SdamLogFilename**
+     - {+not-available+}
+     - | The name of the SDAM (Server Discovery and Monitoring) log file.
+       |
+       | **Data Type**: {+string-data-type+}
+       | **Default**: ``null``
+     
+   * - **Server**
+     - ``mongodb://sample.host:27017``
+     - | The host and port number where MongoDB is running.
+       |
+       | **Data Type**: ``MongoServerAddress``
+       | **Default**: ``localhost:27017``
+
+   * - **ServerApi**
+     - {+not-available+}
+     - | Allows opting into Stable API versioning. See 
+         :manual:`the MongoDB Manual</reference/stable-api>` for more information about
+         Stable API versioning.
+       |
+       | **Data Type**: ``ServerApi``
+       | **Default**: ``null``
+
+   * - **Servers**
+     - .. code-block:: none
+        :copyable: false
+     
+        mongodb://sample.host1:27017,
+        sample.host2:27017,
+        sample.host3:27017
+     - | The cluster members where MongoDB is running. 
+       |
+       | **Data Type**: ``List<MongoServerAddress>``
+       | **Default**: ``localhost:27017``
+     
+   * - **ServerSelectionTimeout**
+     - ``serverSelectionTimeoutMS=15000``
+     - | The length of time the driver tries to select a server before timing out.
+       |
+       | **Data Type**: ``TimeSpan``
+       | **Default**: 30 seconds
+
+   * - **SocketTimeout**
+     - ``socketTimeoutMS=0``
+     - | The length of time the driver tries to send or receive on a socket before 
+         timing out. Specifying ``0`` means no timeout.
+       |
+       | **Data Type**: ``TimeSpan``
+       | **Default**: OS default
+
+   * - **SrvMaxHosts**
+     - ``srvMaxHosts=3``
+     - | The greatest number of SRV results to randomly select when initially populating 
+         the seedlist or, during SRV polling, adding new hosts to the topology. 
+         Specifying ``0`` means no maximum.
+       |
+       | **Data Type**: {+int-data-type+}
+       | **Default**: ``0``
+
+   * - **SslSettings**
+     - ``tlsDisableCertificateRevocationCheck=false``
+     - | TLS/SSL options, including client certificates, revocation handling, and 
+         enabled and disabled TLS/SSL protocols.
+       |
+       | **Data Type**: ``SslSettings``
+       | **Default**: ``null``
+
+   * - **UseTls**
+     - ``tls=true`` or ``ssl=true``
+     - | Specifies whether to require TLS for connections to the server. If you use 
+         a scheme of ``\"mongodb+srv\"`` or specify other TLS options, 
+         this option defaults to ``true``.
+       |
+       | **Data Type**: {+bool-data-type+}
+       | **Default**: ``false``
+
+   * - **WaitQueueTimeout**
+     - ``waitQueueTimeoutMS=0``
+     - | The length of time the driver tries to check out a connection from a 
+         server's connection pool before timing out. Specifying ``0`` means no timeout.
```

Comment:
Do we want users to use `TimeSpan.Infinite` instead of `0`? If so, I'll remove this sentence from the docs. I only included it because I thought it was unintuitive and wanted people to be aware.

Reply:
Looks good.